### PR TITLE
Branch update user guide

### PR DESCRIPTION
**Pull request by EkChinHui**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -564,31 +568,39 @@ This will edit an expenditure at **index 1** to be **renamed** "Long-Sleeve" wit
...
+
+You can use this command to **quickly find your expenditures**.
+
+Use this command to filter through your expenditures by keywords.
+NUSave will display all expenditures whose title contains the entered keyword/key phrase.
 
    ```📕 Note: Expenditures will be filtered as long as they contain the search term. The filter is case-insensitive, using upper-case or lower-case will not make a difference in the search result.```
 
-> ⚠️ **You can only use this command if there are existing expenditures**.
+> ⚠️ **You should not use this command if budget page is empty**.
```
Comment (wenhaogoh) [inline docs/UserGuide.md]: Don't think you should use bold here as the symbol is sufficient. It is also more consistent with the other commands.

**Review by wenhaogoh — COMMENTED** [review]
Just some nits to fix

File: docs/UserGuide.md (hunk 1 above)
Comment (EkChinHui) [reply docs/UserGuide.md]: Good spot! I'll change it ASAP. 😄

**Review by wenhaogoh — APPROVED** [review]
LGTM

**Pull request merged**